ci/scripts/build_all_examples.sh

```diff
@@ -39,7 +48,18 @@ function build_all_configs() {
                 continue
             fi
             
-            build_example "$example" "$sdk_version" "$board" "$sd_variant" "$toolchain" "$config_dir" "$build_dir" "$log_level" || exit 1
+            local start_ts=`date +%s%N`
```

Comment:
Nanoseconds precision is not supported on the MacOS. We could use this variant for it: `date +%s000000000`.